client/me/privacy/main.jsx

```diff
@@ -102,7 +102,7 @@ const Privacy = createReactClass( {
 				<DocumentHead title={ translate( 'Privacy Settings' ) } />
 				<MeSidebarNavigation />
 				<ReauthRequired twoStepAuthorization={ twoStepAuthorization } />
-				<SectionHeader label={ translate( 'Usage Information' ) } />
+				<SectionHeader label={ translate( 'Usage information' ) } />
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.